front/public/locales/en/errors.json

```
@@ -1,11 +1,175 @@
+            "UnparsableErrorOutput":"Core returned an error in an unknown format"
+        },
+        "document": {
+            "NotFound":"Document '{{document_key}}' not found"
+        },
+        "electrical_profiles": {
+            "NotFound":"Electrical Profile Set '{{electrical_profile_set_id}}', could not be found"
+        },
+        "geometry": {
+            "UnexpectedGeometry":"expected geometry {{expected}} but got {{actual}}"
+        },
+        "infra_cache": {
+            "ObjectNotFound":"{{obj_type}} '{{obj_id}}', could not be found everywhere in the infrastructure cache"
+        },
+        "infra": {
+            "NotFound":"",
+            "edition": {
+                "InfraIsLocked":"Infra is locked"
+            },
+            "errors": {
+                "WrongErrorTypeProvided":"Wrong Error type provided"
+            },
+            "lines": {
+                "LineNotFound":"no line with code {{line_code}} found"
+            },
+            "objects": {
+                "DuplicateIdsProvided":"Duplicate object ids provided",
+                "ObjectIdNotFound": "L'objet n'a pas été trouvé"
+            },
+            "pathfinding": {
+                "EndingTrackLocationNotFound":"Ending track location was not found",
+                "InvalidNumberOfPaths":"The pathfinding cannot return 5 paths (expected: [1-5])",
+                "StartingTrackLocationNotFound":"Starting track location was not found"
+            },
+            "railjson": {
+                "WrongRailjsonVersionProvided":"Wrong Railjson version provided"
+            }
+        },
+        "layers": {
+            "LayerNotFound":"Layer {{layer_name}} not found. Expected one of {{expected_name",
+            "ViewNotFound":"View {{view_name}} not found. Expected one of {{expected_names}}"
+        },
+        "operation": {
+            "EmptyId":"Empty string id is forbidden",
+            "InvalidPatch":"A Json Patch error occurred",
+            "ModifyId":"Update operation try to modify object id, which is forbidden",
+            "ObjectNotFound":"Object '{{obj_id}}', could not be found in the infrastructure '{{infra_id}}'"
+        },
+        "pagination": {
+            "InvalidPage": "Invalid page number ({{page}})",
+            "InvalidPageSize":"Invalid page size ({{provided_page_size}}), expected an integer 0 < page_size <= {{max_page_size}}"
+        },
+        "pathfinding": {
+            "ElectricalProfilesOverlap": "Electrical Profile overlaps with others",
+            "ElectrificationOverlap":"Electrification {{electrification_id}} overlaps with other electrifications",
+            "InfraNotFound":"Infra {{infra_id}} does not exist",
+            "NotFound":"Pathfinding {{pathfinding_id}} does not exist",
+            "OperationalPointsNotFound":"Operational points do not exist: {{operational_points}}",
+            "RollingStockNotFound":"Rolling stock with id {{rolling_stock_id}} doesn't exist",
```

To harmonize with you previous trads
```suggestion
            "RollingStockNotFound":"Rolling stock with id {{rolling_stock_id}} does not exist",
```